common/lib/dependabot/pull_request_creator.rb

```diff
@@ -248,7 +248,8 @@ def labeler
           includes_security_fixes: includes_security_fixes?,
```

Comment:
Technically this isn't needed since it can be inferred from `vulnerabilities` fixed, but I did not want to add backwards incompatible changes by removing it